kubernetes/hands-on-lab/domain.yaml

```diff
@@ -0,0 +1,126 @@
+# Copyright 2017, 2019, Oracle Corporation and/or its affiliates. All rights reserved.
```

Comment:
You need to fix the Copyright
Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.

Reply:
I changed the Copyright date